lib/trino-orc/src/main/java/io/trino/orc/OrcWriterOptions.java

```diff
@@ -27,6 +27,24 @@
 
 public class OrcWriterOptions
 {
+    public enum WriterIdentification
+    {
+        /**
+         * Write ORC files with a writer identification and version number that is readable by Hive 2.0.0 to 2.2.0
+         */
+        LEGACY_HIVE_COMPATIBLE,
+
+        /**
+         * Write ORC files with the legacy writer identification of PrestoSQL
+         */
+        PRESTO,
```

Comment:
Can we actually do that (have presto in codebase)?

Reply:
we have in other places in the code. But no problem to change it in the future if need arises.